internal/slicer/slicer_test.go

```diff
@@ -1478,6 +1478,270 @@ var slicerTests = []slicerTest{{
 		`,
 	},
 	error: `cannot find package "test-package" in archive\(s\)`,
+}, {
+	summary: "Valid hard link in two slices in the same package",
+	slices: []setup.SliceKey{
+		{"test-package", "slice1"},
+		{"test-package", "slice2"}},
+	pkgs: []*testutil.TestPackage{{
+		Name: "test-package",
+		Data: testutil.MustMakeDeb([]testutil.TarEntry{
+			testutil.Dir(0755, "./"),
+			testutil.Dir(0755, "./"),
+			testutil.Reg(0644, "./file", "foo"),
+			testutil.Hlk(0644, "./hardlink", "./file"),
+		}),
+	}},
+	release: map[string]string{
+		"slices/mydir/test-package.yaml": `
+		package: test-package
+		slices:
+			slice1:
+				contents:
+					/**:
```

Comment:
Can we add the files explicitly in this case please? In this case it is relevant because `/hardlink` is in both.